HelloVectors/HelloVectors_win32_cpp/LottieTool/SquareCircleMorph.h

```diff
@@ -14,40 +14,33 @@ using namespace Windows::Foundation::Numerics;
 
 #include "ICompositionSource.h"
 
+namespace Microsoft::UI::Xaml::Controls {
```

Comment:
@kennykerr this doesn't work currently, i'm trying to define IAnimatedVisual locally, rather than needing a reference to the XAML nuget which contains the winmd where the interface lives, reason being this project is raw win32 with no XAML.  Can't quite figure out how to make this compile.. is there an equivalent to winrt::implements for defining interfaces?

Reply:
You can define the ABI directly, but defining the projected interface is not easy. Alternatively if you have an IDL file handy, you can redefine the interface and have cppwinrt generate the scaffolding for you.